CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,11 @@
+# Contributor Code of Conduct
```

Comment:
we have uber approved `CODE_OF_CONDUCT`

https://github.com/uber-go/zap/blob/master/CODE_OF_CONDUCT.md